setup/paths.sh

```diff
@@ -37,38 +37,27 @@ else
     source $BUILD_HOME/cmsgemos/setup/paths.sh
 fi
 
+# Export project
+export GEM_PLOTTING_PROJECT=$BUILD_HOME/gem-plotting-tools
+
+# Setup Path
+export PATH=$PATH:$GEM_PLOTTING_PROJECT
+export PATH=$PATH:$GEM_PLOTTING_PROJECT/macros
+export PATH=$PATH:$GEM_PLOTTING_PROJECT/setup
+
+# Setup PYTHONPATH
+export PYTHONPATH=$PYTHONPATH:$BUILD_HOME/vfatqc-python-scripts
+export PYTHONPATH=$PYTHONPATH:$GEM_PLOTTING_PROJECT
+export PYTHONPATH=$PYTHONPATH:$GEM_PLOTTING_PROJECT/macros
+export PYTHONPATH=$PYTHONPATH:$GEM_PLOTTING_PROJECT/setup
```

Comment:
Sure we can implement these changes.  This is a good idea and will increase readability/maintenance in the long turn.